### PR TITLE
Add shiden network

### PR DESCRIPTION
**Pull request by hskang9**
Aggregates PR #388

### CODE REVIEW

**Comment by hskang9** [comment]
@ChaituVR

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: src/networks.json [hunk 1]
```diff
@@ -554,15 +564,5 @@
...
-  },
-    "44": {
-    "key": "44",
-    "name": "Crab Network",
-    "shortName": "Crab",
-    "chainId": 44,
-    "network": "mainnet",
-    "multicall": "0x4617D470F847Ce166019d19a7944049ebB017400",
-    "rpc": ["https://crab-rpc.darwinia.network"],
-    "explorer": "https://crab.subscan.io/"
```
[the remark this answers is not in the document]
Comment (hskang9) [reply src/networks.json:566]: I added crab

File: src/networks.json [hunk 2]
```diff
@@ -554,15 +574,5 @@
     "multicall": "0x6F522a3982e8F9A50A2EDc9E46ed1A3aE2B3FD3a",
     "rpc": ["https://evm-cronos.crypto.org"],
     "explorer": "https://cronos.crypto.org/explorer"
-  },
```
Comment (bonustrack) [inline src/networks.json:557]: Could you remove changes here?

Comment (hskang9) [reply src/networks.json:557]: ok

Comment (hskang9) [reply src/networks.json:557]: @bonustrack I changed the code, can you check?

**Comment by ChaituVR** [comment]
@hskang9 You accidentally removed some other network 
image

This PR should only add new network

**Comment by hskang9** [comment]
I changed codebase to recent addition, and added shiden

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Please revert changes in package-lock.json and src/networks/256.png

**Pull request closed without merging**